packages/code-du-travail-api/src/server/routes/api.js

```diff
@@ -70,24 +70,107 @@ router.get(`${BASE_URL}/suggest`, async ctx => {
  * @returns {Object} Result.
  */
 router.get(`${BASE_URL}/items/:source/:slug`, async ctx => {
-  ctx.body = await codeDuTravailNumerique.getSingleItem({
+  const item = await codeDuTravailNumerique.getSingleItem({
     source: ctx.params.source,
     slug: ctx.params.slug,
     _source: [
       "title",
       "url",
       "html",
       "date_debut", // code-du-travail
+      "date_fin", // code-du-travail
       "date",
       "path", // code-du-travail
       "id", // idcc, kali
+      "tags", // code-du-travail
       "description", // modele de courrier
       "filename", // filename
       "author" // faq
     ]
   });
+  // console.log(items)
+
+  let relatedItems = {};
```

Comment:
A `let` pattern that could be a `const` if equal to a function

```js
const relatedItems = ctx.params.source === "faq" ? getRelatedItemsFromItem(themes, item) : {};
```

(`getRelatedItemsFromItem` might not be the best function name here)

Reply:
`const` is easier to stack in my brain that `let`.
When I see a `let` I'm always looking for where the variable is modified.
When it's a `const` I don't have to do it :+1: